lib/commands/general.js

```diff
@@ -124,8 +124,10 @@ commands.background = async function (seconds) {
   if (this.opts.startActivityArgs && this.opts.startActivityArgs[`${appPackage}/${appActivity}`]) {
     // the activity was started with `startActivity`, so use those args to restart
     args = this.opts.startActivityArgs[`${appPackage}/${appActivity}`];
-  } else if (appPackage === this.opts.appPackage && appActivity === this.opts.appActivity ||
-    appPackage === this.opts.appWaitPackage && this.opts.appWaitActivity.split(',').indexOf(appActivity) >= 0) {
+  } else if ((appPackage === this.opts.appPackage && appActivity === this.opts.appActivity) ||
+    (appPackage === this.opts.appWaitPackage &&
+      this.opts.appWaitActivity &&
```

Comment:
you could use `(this.opts.appWaitActivity || '').split(',').includes(appActivity)` instead